src/opening-period/page/CreateNewOpeningPeriodPage.test.tsx

```diff
@@ -171,6 +171,19 @@ describe(`<CreateNewOpeningPeriodPage />`, () => {
           },
         ],
       },
+      timeSpanGroup: {
```

Comment:
Should we add a unit test for submitting opening period with (multiple?) rules also? The rendering of those rules seems to be tested in the editOpeningPeriodPage.test.tsx, but I couldn't find a unit test for submitting the form with time span rules? I wonder if it would be good to have, though at the same time I know unit testing this form has  been hard, as we have had problems with timeouts in them?

Reply:
I could actually extend the current test 🤔

Reply:
I think simulating user interactions makes testing hard. It would be nicer to test this by populating the fields and them sent them back to server. I guess this test already covers it for edit.
But if we want to test it for save also it requires more work.